goal_src/jak3/build/all_objs.json

```diff
@@ -0,0 +1,2764 @@
+["hover-enemy-h",                                   "hover-enemy-h",                                   5, ["TEMA", "SEA", "FACC", "LFORM", "FACD", "LPATK", "TOWERA", "PRECA", "VOCA"], ""],
+["hover-nav-control",                               "hover-nav-control",                               5, ["TEMA", "SEA", "FACC", "LFORM", "FACD", "LPATK", "TOWERA", "PRECA", "VOCA"], ""],
+["hover-enemy",                                     "hover-enemy",                                     5, ["TEMA", "SEA", "FACC", "LFORM", "FACD", "LPATK", "TOWERA", "PRECA", "VOCA"], ""],
+["hover-formation",                                 "hover-formation",                                 5, ["TEMA", "SEA", "FACC", "LFORM", "FACD", "LPATK", "TOWERA", "PRECA", "VOCA"], ""],
+["hover-nav-edit",                                  "hover-nav-edit",                                  5, ["TEMA", "SEA", "FACC", "LFORM", "FACD", "LPATK", "TOWERA", "PRECA", "VOCA"], ""],
+["dp-bipedal-part",                                 "dp-bipedal-part",                                 5, ["TEMA", "LFORM", "FACD", "LBIPED", "LPATK", "TOWERA", "PRECA"], ""],
+["dp-bipedal-shot",                                 "dp-bipedal-shot",                                 5, ["TEMA", "LFORM", "FACD", "LBIPED", "LPATK", "TOWERA", "PRECA"], ""],
+["dp-bipedal-FACD-LBIPED-LFORM-LPATK-PRECA-TEMA-TOWERA", "dp-bipedal",                                      5, ["TEMA", "LFORM", "FACD", "LBIPED", "LPATK", "TOWERA", "PRECA"], ""],
+["dp-bipedal-FACD-LBIPED-LFORM-LPATK-PRECA-TEMA-TOWERA", "dp-bipedal",                                      5, ["TEMA", "LFORM", "FACD", "LBIPED", "LPATK", "TOWERA", "PRECA"], ""],
+["hover-nav-templea",                               "hover-nav-templea",                               5, ["TEMA"], ""],
+["temple-part",                                     "temple-part",                                     5, ["TEMA"], ""],
+["temple-obs",                                      "temple-obs",                                      5, ["TEMA"], ""],
+["temple-scenes",                                   "temple-scenes",                                   5, ["TEMA"], ""],
+["temple-obs2",                                     "temple-obs2",                                     5, ["TEMA"], ""],
+["temple-mood",                                     "temple-mood",                                     5, ["TEMA"], ""],
+["templea-texture",                                 "templea-texture",                                 5, ["TEMA"], ""],
+["hover-training",                                  "hover-training",                                  5, ["TEMA"], ""],
+["tomb-baby-spider-TEMA",                           "tomb-baby-spider",                                5, ["TEMA"], ""],
+["tomb-baby-spider-TEMA",                           "tomb-baby-spider",                                5, ["TEMA"], ""],
```

Comment:
We still need to do something about this - can you detect if the type of a data-only object file is a child of (or exactly) `art-group`?